editor/modes/visual-editor/block.js

```diff
@@ -364,6 +376,19 @@ class VisualEditorBlock extends Component {
 						<div className="editor-visual-editor__block-controls">
 							<BlockSwitcher uid={ block.uid } />
 							<Slot name="Formatting.Toolbar" />
+							<Toolbar className="editor-visual-editor__mobile-tools">
+								{ ( showUI || isHovered ) && <BlockMover uids={ [ block.uid ] } /> }
```

Comment:
Tried this but could not get it to work on mobile reliably between proper flex positioning, sticky position, etc.